src/main/java/com/sap/sgs/phosphor/fosstars/model/value/Language.java

```diff
@@ -0,0 +1,47 @@
+package com.sap.sgs.phosphor.fosstars.model.value;
+
+import java.util.Objects;
+
+/**
+ * The enum contains programming languages.
+ */
+public enum Language {
+
+  C, CPP,
+
+  // JVM languages
```

Comment:
Kotlin? As this is a well known language

Reply:
Sure. To be honest, I didn't want to list all well-known languages. The enum contains languages from the following categories:

- memory-unsafe languages
- languages covered by security alerts on GitHub
- and Java :)

Those languages are going to be used soon in several scores. Let's add other languages when we need them.